### DISCUSSION TITLE
Should we rename `master` to `main` to match the TFX Addons pattern?

### DISCUSSION THREAD

**Discussion by hanneshapke** [tensorflow/model-card-toolkit#236] [Ideas]
To match the pattern from tfx-addons.

Why? 
https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main

**Comment by codesue** [comment]
Let's do it! It seems pretty feasible since GitHub does a lot to handle redirects and there aren't too many mentions to update in the codebase: https://github.com/tensorflow/model-card-toolkit/search?p=1&q=master.

**Comment by codesue** [comment]
Completed in https://github.com/tensorflow/model-card-toolkit/pull/260
